examples/site/howto_local_development/README.md

```diff
@@ -34,25 +34,86 @@ would use:
 
 ```shell
 cd $HOME
-git clone https://github.com/microsoft/vcpkg
-(cd vcpkg && ./bootstrap.sh)
+git clone -q https://github.com/microsoft/vcpkg
+# Expected output: none
+./vcpkg/bootstrap-vcpkg.sh --disableMetrics --useSystemBinaries
```

Comment:
I see this:

```
 ./vcpkg/bootstrap-vcpkg.sh --disableMetrics --useSystemBinaries

Could not find ninja. Please install it (and other dependencies) with:
sudo apt-get install cmake ninja-build
```

Reply:
It seems to work (before) and despite this error

Reply:
Okay, the `--useSystemBinaries` flag is a bad idea, removed.  That adds some more output (the first time your run this), so I added that to the expected stuff.